src/components/Input.scss

```diff
@@ -158,5 +159,39 @@ $form-input-bg-color: $cu-foreground--dark !default;
       @include ss-pika;
     }
   }
+}
 
+// * 5. Errors and feedback
+.Input-feedback {
+  display: none;
+}
+.is-invalid {
+  &.is-required {
+    .Input-label:after {
+      color: $cu-negative;
+    }
+  }
+  .Input-element {
+    border-color: $cu-negative;
+  }
+  .Input-feedback {
+    display: block;
+    color: $cu-negative;
+  }
 }
+// should this be moved to its own scss file since it will also apply to checkboxes/radios/selects?
+// should there be a "forms" component with examples like this and its own scss?
+.is-valid {
```

Comment:
We should probably style the different form components individually with error states. If we want to provide a more generic treatment this could be added via some sort of `FormControl` wrapper.